platforms/json_converter.py

```diff
@@ -0,0 +1,102 @@
+import requests, os, datetime
```

Comment:
According to PEP-8 coding standards, you should write every individual import in separate lines. https://www.python.org/dev/peps/pep-0008/#imports